cupyx/scipy/signal/_bsplines.py

```diff
@@ -393,3 +406,111 @@ def qspline1d_eval(cj, newx, dx=1.0, x0=0):
         result += cj[indj] * _quadratic(newx - thisj)
     res[cond3] = result
     return res
+
+
+def cspline2d(signal, lamb=0.0, precision=-1.0):
```

Comment:
Seems that scipy uses `lambda` for the name of the second argument.  Any reason to choose `lamb` here?
https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.cspline2d.html

Reply:
As `lambda` is a reserved keyword in Python, then it is not possible to use it. Since SciPy is generating the signature from C, I think it would be possible to declare lambda as a parameter. However in the practice, any use of `lambda` will trigger a `SyntaxError`. https://github.com/scipy/scipy/blob/897d811c616ec380df4000e117034e0c4cb2815c/scipy/signal/_splinemodule.c#L22

This is a bug in SciPy